server/src/collections/collections.service.ts

```diff
@@ -98,6 +102,32 @@ export class CollectionsService {
     return data;
   }
 
+  /**
+   * Get collections statistics data
+   * @returns {collectionCount:number, totalData:number}
+   */
+  async getStatistics() {
+    const data = {
```

Comment:
it is not a const, please use let instead

Reply:
![image](https://user-images.githubusercontent.com/20559208/123383488-d0020900-d5c5-11eb-9653-50043e110f61.png)